plugins/modules/python_repository.py

```diff
@@ -89,13 +95,16 @@ def main():
         argument_spec={
             "name": {},
             "description": {},
+            "autopublish": {"type": "bool", "default": False},
```

Comment:
I think the default here is actually dangerous.
Do you feel like adding this flag to a test to demonstrate not specifying it on the second invocation does not reset it?

Reply:
You are right. Setting a default for `autopublish` could cause unwanted repo changes. I'll update my branch accordingly (probably not today).

Reply:
The change works as intended in my Ansible play. However, I need to dig into the tests before I can add flag checking.